.github/workflows/issue-autorespond-and-close.yml

```
@@ -0,0 +1,14 @@
+name: Auto respond and close issue
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  invoke_reusable_workflow:
+    uses: mparticle/mparticle-workflows/.github/workflows/auto-respond-close-issue.yml@master
```

This should be either `stable` or `main`.

```suggestion
    uses: mparticle/mparticle-workflows/.github/workflows/auto-respond-close-issue.yml@main
```